docs/user-guide/dream/dream-instrument-view.ipynb

```diff
@@ -229,6 +229,65 @@
    "source": [
     "dream.instrument_view(dg[\"endcap_backward\"][\"module\", 0].hist(tof=1))"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "15",
+   "metadata": {},
+   "source": [
+    "## Display inhomogeneity from absorption\n",
```

Comment:
Nice way of visualizing, but I am not sure this is the right notebook to put it in?
It feels sort of unrelated to the rest?

Would it be better as part of something like a gallery notebook (even though we don't have a gallery yet?).
Or even keep a list of 'recipes' in e.g. our github discussions (and refer to them in the docs)?
This visualization sounds like it is something that the instrument scientist would find useful just for themselves, but isn't a necessary part of a reduction workflow?

Reply:
Sure, we can put it in a separate notebook.

It's mainly useful for IS to estimate what kind of impact absorption has in some experiment setup.
Celine mentioned that this would be good step towards having an automatic absorption correction.
Making absorption correction part of the workflow is started in https://github.com/scipp/essdiffraction/pull/104/